.github/.OwlBot.yaml

```diff
@@ -4,6 +4,7 @@ docker:
 deep-remove-regex:
   - /accessapproval/apiv1/
   - /accesscontextmanager/apiv1/
+  - /ai/generativelanguage/v1beta2/
```

Comment:
across this and the other file missing api in front of the version.

Reply:
Done, thanks!